src/components/Recorder/Recorder.tsx

```diff
@@ -200,10 +222,9 @@ export class Recorder extends Component<Props, State> {
                 {!video ? (
                     <div className={`${styles["express-recorder__recorder__no-video"]}`}>
                         <div class={`xr_no-video-text ${styles["no-video-text"]}`}>
-                            Recording Audio Only
-                            {stream && (
+                            {screenShareOn && (
```

Comment:
```suggestion
                            {screenShareOn ? (< />) : null}
```
in this case, `screenShareOn` is a boolean. so let's make sure that we don't print out any "false" to the screen.